### PR TITLE
use look aside cache for downstream

### PR DESCRIPTION
**Pull request by jcantrill**
Use look aside cache for downstream. Validating downstream build: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=15698191

### CODE REVIEW

File: kibana/Dockerfile [hunk 1]
```diff
@@ -1,45 +1,70 @@
+#oit## This file is managed by the OpenShift Image Tool: https://github.com/openshift/enterprise-images
+#oit## by the OpenShift Continuous Delivery team (#aos-cd-team on IRC).
+#oit## 
+#oit## Any yum repos listed in this file will effectively be ignored during CD builds.
+#oit## Yum repos must be enabled in the oit configuration files.
+#oit## Some aspects of this file may be managed programmatically. For example, the image name, labels (version,
+#oit## release, and other), and the base FROM. Changes made directly in distgit may be lost during the next
+#oit## reconciliation.
+#oit##
```
Comment (richm) [inline kibana/Dockerfile]: please omit this

**Comment by richm** [comment]
```
[WARNING] Logging test suite test-access-control failed at Thu Apr  5 13:44:36 UTC 2018
```

**Comment by ewolinetz** [comment]
strange that the logging test passed...
/retest

**Pull request merged**